### DISCUSSION TITLE
Any plans for a public release of 2.2.x?

### DISCUSSION THREAD

**Discussion by toge** [zlib-ng/zlib-ng#1755] [Q&A] [answered]
Thanks a lot for the significant improvement in zlib-ng performance in 2.2.0!
While I want to support 2.2.x in [conan's recipe](https://conan.io/center/recipes/zlib-ng), it's status is release candidate.
Are there any plans for public release?

**Comment by Dead2** [comment, marked as answer]
@toge Yes, #1753 is the pending 2.2.1 stable release and will be released within days unless anyone finds a bug that warrants a delay.

**Comment by toge** [comment]
@Dead2
Thanks for the quick response.
I look forward to the release of 2.2.1!
